taxi/views.py

```diff
@@ -61,10 +68,18 @@ class CarListView(LoginRequiredMixin, generic.ListView):
 class CarDetailView(LoginRequiredMixin, generic.DetailView):
```

Comment:
`   
 model = Driver
    form_class = DriverCreateForm
    success_url = reverse_lazy("taxi:driver-list")
`

you can do it like this so it would be more readable (not sure tho)